ui/apps/platform/src/Containers/Administration/Events/AdministrationEventsEmptyState.tsx

```diff
@@ -17,12 +25,25 @@ function AdministrationEventsEmptyState({
             <Tr>
                 <Td colSpan={colSpan}>
                     <Bullseye>
-                        <EmptyState>
-                            {hasFilter && <EmptyStateIcon icon={FilterIcon} />}
-                            <EmptyStateBody>
-                                {hasFilter ? 'No events found' : 'No events'}
-                            </EmptyStateBody>
-                        </EmptyState>
+                        {hasFilter ? (
+                            <EmptyState>
+                                <EmptyStateIcon icon={SearchIcon} />
+                                <EmptyStateBody>
+                                    <Flex direction={{ default: 'column' }}>
+                                        <Title headingLevel="h2">
+                                            No administration events found
+                                        </Title>
+                                        <Text>Modify filters and try again</Text>
+                                    </Flex>
+                                </EmptyStateBody>
+                            </EmptyState>
+                        ) : (
+                            <EmptyState>
+                                <EmptyStateBody>
+                                    <Title headingLevel="h2">No administration events</Title>
+                                </EmptyStateBody>
+                            </EmptyState>
+                        )}
```

Comment:
This is a good idea and probably is a pattern we should replicate elsewhere.

Reply:
Could we expand the use of this wrapper component I created a write back, and use it in most cases like this?

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/PatternFly/EmptyStateTemplate/EmptyStateTemplate.tsx

Reply:
The default value for icon was the problem. Allow no prop to mean no icon, and we can reuse here.
* CubesIcon represents Kubernetes pods according to guidelines.
    https://www.patternfly.org/design-foundations/icons
* I could not find any icon that represents no things generically, pardon the double negative.

Reply:
I'm fine with making the icon for that component optional. (I created it very early in our PatternFly journey, and the docs for their <EmptyState> icon are unclear on what the sample icon was for.)

Reply:
You are gracious <3

I will do as follow up and as bonus will also ask about guideline at PatternFly Design Share next week.